salesagility/SuiteCRM
# Fix PDF structure

## Description




- Renames MPDF to LegacyMPDF.
- Adds labels to the PDF engines for the PDF settings page.
- Adds PDF extension using extension framework.

## How To Test This

1. Check that MPDF + TCPDF still work as expected.
2. Check that MPDF and TCPDF show the correct labels on the PDF settings page.

## Types of changes


### Final checklist
